docs/commcare-cloud/commands/index.md

```diff
@@ -566,8 +566,7 @@ tmux windows will a new one be opened.
 Export Sentry events. One line per event JSON.
 
 ```
-commcare-cloud <env> export-sentry-events -k API_KEY -p PROJECT_ID [-q QUERY] [--start START] [--end END]
-                                          [--organization ORGANIZATION] [--full]
+commcare-cloud <env> export-sentry-events -k API_KEY -i ISSUE_ID [--full] [--cursor CURSOR]
```

Comment:
Were the changes in this file committed by mistake?  They seem unrelated to the rest of the PR.

Reply:
I'm guessing this has to do with an external change from another PR? If the build is failing it means that the files match what `make` makes

Reply:
I'll merge and then test it locally to make sure, and make a PR with any touch ups necessary